index.js

```diff
@@ -4,12 +4,29 @@
 module.exports = {
     name: 'ember-cli-hot-loader',
     serverMiddleware: function (config){
-        var lsReloader = require('./lib/hot-reloader')(config.options);
-        lsReloader.run();
+        if (config.options.environment === 'development') {
```

Comment:
Actually, I think that a better default is that if we're watching (running `ember b -w` or `ember s`) we should enable the hot-loader. For static builds we shouldn't. Anyway, we can bikeshed that separate. Not a blocker.